util/example/tlgen/xbar_main.hjson

```diff
@@ -2,82 +2,187 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 { name: "main",
-  clock: "main", // Main clock, used in sockets
+  type: "xbar",
```

Comment:
do you think longer term its worth showing an embedded xbar example? Similar to the one you just added? Maybe even the non-contiguous address requirement like you had already showed.

Reply:
Hum... That is also good. I will add xbar device/host port to another example. :) Thanks!